.github/workflows/tests.yml

```diff
@@ -84,11 +84,13 @@ jobs:
         if: env.GIT_DIFF
         run: |
           EXPERIMENTAL=true make test-unit-cover
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v3.1.1
```

Comment:
I think if we use `v3` then it will take the most recent version.

Reply:
OK